packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_notifications/blocs/notifications_bloc_extensions.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+part of 'notifications_bloc.dart';
+
+/// Extensions for the NotificationsBloc
+extension _SendNotificationsBlocExtensions on NotificationsBloc {
+  // Sends a push message to the remote server
+  Future<bool> _sendMessage(_SendMessageEventArgs args) async {
```

Comment:
Not sure what you mean about having two extensions with same name? Which is the second one you are referring to?

Reply:
In this file we have 
Future<bool> _sendMessage(_SendMessageEventArgs args)
and
Stream<Result<bool>> _sendMessage(NotificationsBloc bloc), that calls the first one. It could be a good practice, but I found it confusing.

Reply:
Ah, I see. Actually, you are right. This should be renamed for clarity. 
Fixed!